snap/snapcraft.yaml

```diff
@@ -1,36 +1,56 @@
-name: saber
-version: 0.9.4
-summary: Handwritten notes
-description: A cross-platform libre handwritten notes app, available for Android, iOS, Linux, Windows, and macOS!
+name: saber # you probably want to 'snapcraft register <name>'
+base: core18 # the base snap is the execution environment for this snap
+version: '0.9.4' # just for humans, typically '1.2+git' or '1.3.2'
+summary: A cross-platform libre handwritten notes app # 79 char long summary
+description: |
+  A cross-platform libre handwritten notes app, available for Android, iOS, Linux, Windows, and the web!
+grade: stable # must be 'stable' to release into candidate/stable channels
```

Comment:
Since we're not at Saber v1.0.0 yet (as per semantic versioning), I'd prefer this as `grade: devel`. Would `devel` cause any complications?

Reply:
No but that's not a big thing, from next updates, I'll release in beta or candidate channel only simple.

Reply:
Wouldn't that leave users who've currently installed the stable channel without updates?